### ISSUE TITLE
TypeError: logger.create is not a function

### ISSUE THREAD

**Issue by dawidow** [wahyubucil/adonis-mix-asset#1]
Hello. Every time (I tried 3 new projects) when I try to execute **invoke** command the program throws such a bug and laravel-mix does not work. How to fix it?

```js
❯ node ace invoke adonis-mix-asset

TypeError: logger.create is not a function


1 instructions
  /Users/dawidowczarz/Desktop/Projekty/devcasts/node_modules/adonis-mix-asset/build/instructions.js:12

2 Instructions.runInstructions
  /Users/dawidowczarz/Desktop/Projekty/devcasts/node_modules/@adonisjs/sink/src/Tasks/Instructions/index.js:265

3 Instructions.execute
  /Users/dawidowczarz/Desktop/Projekty/devcasts/node_modules/@adonisjs/sink/src/Tasks/Instructions/index.js:325

4 Invoke.run
  /Users/dawidowczarz/Desktop/Projekty/devcasts/node_modules/@adonisjs/assembler/build/commands/Invoke.js:50

5 async Invoke.exec
  /Users/dawidowczarz/Desktop/Projekty/devcasts/node_modules/@adonisjs/ace/build/src/BaseCommand/index.js:136

6 async Kernel.runCommand
  /Users/dawidowczarz/Desktop/Projekty/devcasts/node_modules/@adonisjs/ace/build/src/Kernel/index.js:238

7 async App.handle
  /Users/dawidowczarz/Desktop/Projekty/devcasts/node_modules/@adonisjs/core/build/src/Ignitor/Ace/App/index.js:307

8 async Ace.handle
  /Users/dawidowczarz/Desktop/Projekty/devcasts/node_modules/@adonisjs/core/build/src/Ignitor/Ace/index.js:33
```

I will add that I use AdonisJS version 5.

**Comment by wahyubucil** [comment]
I think it's all because you already use the latest AdonisJs core. You can use the `v2.0.0-0` of this package. Install it with `npm i --save-dev adonis-mix-asset@next`

AdonisJs October release (https://preview.adonisjs.com/blog/october-2020-release) doesn't include releasing the new Starter Kit. That's why I will release `v2` of this package when AdonisJs Starter Kit already uses the latest core. And in my opinion, the latest core still not stable enough to use it in production.

If it solves your problem. You can close this issue. Thanks!

**Comment by thetutlage** [comment]
@wahyubucil The latest core has been released and hence it is what gets installed every time you create a new package. I have created a PR to address the migration https://github.com/wahyubucil/adonis-mix-asset/pull/2

**Comment by dawidow** [comment]
@wahyubucil thanks a lot. Now everything works fine. ;)

**Comment by wahyubucil** [comment]
#2 already merged
It's released under the `next` tag on `v2.0.0-1`. Install it with `npm i --save-dev adonis-mix-asset@next`
I will release it under stable if the AdonisJS Web Application scaffold already supported the latest core

Thanks for the help! @thetutlage 
And thanks for the feedback! @dawidow
